src/directives/style-map.ts

```diff
@@ -70,7 +70,8 @@ export const styleMap = directive((styleInfo: StyleInfo) => (part: Part) => {
   for (const name in oldInfo) {
     if (!(name in styleInfo)) {
       if (name.indexOf('-') === -1) {
-        (style as any)[name] = null;
+        // tslint:disable-next-line:no-any
+        (style as any)[name as keyof StyleInfo] = null;
```

Comment:
If you cast style to any, you shouldn't need to cast name as well. Doing it for docs?

Reply:
True, done